server/events/vcs/bitbucketserver/models.go

```diff
@@ -64,3 +64,10 @@ type Changes struct {
 	NextPageStart *int  `json:"nextPageStart,omitempty"`
 	IsLastPage    *bool `json:"isLastPage,omitempty" validate:"required"`
 }
+
+type ConflictStatus struct {
+	CanMerge   *bool    `json:"canMerge,omitempty" validate:"required"`
+	Conflicted *bool    `json:"conflicted,omitempty" validate:"required"`
+	Outcome    *bool    `json:"outcome,omitempty" validate:"required"`
```

Comment:
We don't care about the `Outcome` and `Vetoes` fields so you can omit them from the struct. The json will parse fine.